protected/humhub/components/behaviors/PolymorphicRelation.php

```diff
@@ -27,16 +27,21 @@
  */
 class PolymorphicRelation extends Behavior
 {
-
     /**
      * @var string the class name attribute
      */
-    public $classAttribute = 'object_model';
+    public string $classAttribute = 'object_model';
 
     /**
      * @var string the primary key attribute
      */
-    public $pkAttribute = 'object_id';
+    public string $pkAttribute = 'object_id';
+
+    /**
+     * @var callable|null
+     * @since 1.16
+     */
+    public $onSet;
```

Comment:
Is this used?

Reply:
You're right. This is left-over from the idea to register callbacks here. See further comments where we discuss the `on*` Methods ...